GitUI/Theming/ThemeRepository.cs

```diff
@@ -71,9 +86,19 @@ private IEnumerable<ThemeId> GetUserCustomizedThemeIds()
                 : Enumerable.Empty<ThemeId>();
         }
 
-        private string GetPath(ThemeId id) =>
-            Path.Combine(id.IsBuiltin
-                ? AppDirectory
-                : DataDirectory, id.Name + Extension);
+        private string GetPath(ThemeId id)
```

Comment:
Is it technically possible to have a theme in a portable build that isn't a 'built-in'?

Reply:
Yes, you can still copy an existing theme and adapt that.
Name is set from the filename.

Reply:
In that case, won't we be falling through to `UserThemesDirectory == null` code path?

Reply:
You would copy the theme to the app data directory, user data directory does not exist (really the same as app data).